.github/workflows/_reusable_pr-comment-list-changes.yml

```diff
@@ -46,3 +46,7 @@ jobs:
           pattern: ${{ inputs.pattern }}
           component-name: ${{ inputs.component-name }}
           doc-site-branch : ${{ inputs.doc-site-branch }}
+      - uses: actions/checkout@v4 # Try to fix "action.yml" not found in post action step
```

Comment:
**suggestion:** if we keep this workaround, we should document that it comes from https://github.com/actions/checkout/issues/692#issuecomment-1502203573